### ISSUE TITLE
Support running on Travis-CI's Xcode beta image

### ISSUE THREAD

**Issue by norio-nomura** [realm/SwiftLint#484]
Maybe we should contain `/Application/Xcode-beta.app/…` to `LC_RPATH` into `SwiftLintFramework` for that.
It also needs change on SourceKitten.

https://github.com/realm/SwiftLint/issues/25#issuecomment-180175740

**Comment by jpsim** [comment]
I'd rather use the `xcode-select` version of Xcode, but that's challenging...

**Comment by norio-nomura** [comment]
I pushed PRs.
By applying them, libraries that not bundled in swiftlint by Homebrew will be searched following order:
```sh
/Application/Xcode.app/…
/Application/Xcode-beta.app/…
```
In swiftlint by .pkg will be also searched both. but the order will be depends on the building environment.

**Comment by norio-nomura** [comment]
swiftlint is not affected by `xcode-select`.
Those PRs will only allow launching swiftlint on Travis-CI's Xcode beta image, then the results of linting may be incorrect on Xcode beta.
But, failing launch causes fails on user's CI, that can be avoided by those PRs.

**Comment by jpsim** [comment]
Done in #485. Thanks @norio-nomura

**Comment by norio-nomura** [comment]
:smile:
